src/main/java/com/synopsys/integration/rest/client/IntHttpClient.java

```diff
@@ -194,6 +195,17 @@ public final void logResponseHeaders(HttpResponse response) {
         logHeaders(responseName, response.getAllHeaders());
     }
 
+    public final void logResponseContent(Response response) {
+        if (LogLevel.TRACE == logger.getLogLevel()) {
+            try {
+                String responseContent = response.getContentString();
+                logger.trace("Response content : " + responseContent);
```

Comment:
The only problem with this is if the response is the content of a file like a jar or a binary this logs a bunch of junk. Im also concerned that this will cause issues with the content. I dont think we can read from an input stream multiple times. So we probably can't do this without creating a copy of the stream so that the response content can be read multiple times.